tsl/src/process_utility.c

```diff
@@ -21,10 +22,24 @@ tsl_ddl_command_start(ProcessUtilityArgs *args)
 		DropdbStmt *stmt = castNode(DropdbStmt, args->parsetree);
 		remote_connection_cache_dropped_db_callback(stmt->dbname);
 	}
-
 	dist_ddl_start(args);
 }
 
+/* AlterTableCmds that need tsl side processing invoke this function
+ * we only process AddColumn command right now.
+ */
+void
+tsl_process_altertable_cmd(Hypertable *ht, AlterTableCmd *cmd)
```

Comment:
Am I correct that both arguments are not changed in the function?
```suggestion
tsl_process_altertable_cmd(const Hypertable *ht, const AlterTableCmd *cmd)
```

Reply:
Right now, they do not. But this is supposed to be a more generic function i..e other AlterTablecmds that need tsl side processing would introduce additional if stmts here. So should not force it to be a const.

Reply:
`const` can be removed when the implementation changes. Currently it helps to understand the code.
btw, may be new lines between variable declarations, assertion and code can help for readability.

Reply:
modified.